CHANGELOG.md

```diff
@@ -0,0 +1,17 @@
+<a name="2.0.0-alpha.0"></a>
+# 2.0.0-alpha.0 titanium-octopus (2016-03-15)
+
+### First release of angular2-material!
+
+This inaugural release includes 6 components:
+* [md-button](src/components/button) (buttons and anchors)
+* [md-card](src/components/card)
+* [md-toolbar](src/components/toolbar)
+* [md-sidenav](src/components/sidenav)
+* [md-checkbox](src/components/checkbox) 
+* [md-progress-circle and md-spinner](src/components/progress-circle)
+
+As the alpha process continues, these components will continue to evolve. There *will* be
+breaking changes between alpha releases; the alpha releases are here for people that want an
+early look or who like to live on the edge are are very tolerant of breaking API and behavior 
```

Comment:
Typo - double "are".  Think it should be: 

"who like to live on the edge **and** are very tolerant of breaking API..."